index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome</a>
+      <a href="#projects">Projects</a>
+      <a id="profile-link" target="_blank" href="https://github.com/austync/Augustine-and-Arnaud-Team">My github</a>
+    </nav>
```

Comment:
1. I think a link tag should have been used before the a tag instead of a tag directly.

Reply:
Hi great job with your review. 
- as part of the requirements and best practice, its recommended to 
- [ ] Give very polite reviews. This is because some developers feel stressed by the work. So, giving polite review to their work can help them overcome it
- [ ] Also every time you give a review you have to specify the reason for that review (`why` you think your idea is better)

Reply:
Nice job! but i recommend you use list tag before the link tag. This is because you have a list of items in the nav menu/